packit_service/worker/reporting.py

```diff
@@ -112,6 +116,7 @@ def set_status(
         description: str,
         check_name: str,
         url: str = "",
+        tf_url: str = "",
```

Comment:
There is a reason why you can't have `url` as `None` (Pagure requires `url`, but can be empty string), doesn't apply to `tf_url` though.